bundles/org.openhab.binding.omnikinverter/src/main/java/org/openhab/binding/omnikinverter/OmnikInverter.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.omnikinverter;
```

Comment:
All classes should be in an internal packages: `org.openhab.binding.omnikinverter.internal`